profiles/easy

```diff
@@ -327,6 +327,8 @@ org.freedesktop.home1.resize-home                               auth_admin_keep
 org.freedesktop.home1.update-home                               auth_admin_keep
 # v257 addition (bsc#1233295)
 org.freedesktop.home1.update-home-by-owner                      auth_admin_keep:auth_admin_keep:yes
+# v258 addition (bsc#1250880)
+org.freedesktop.home1.manage-signing-keys                       auth_admin_keep:auth_admin_keep:auth_admin_keep
```

Comment:
I don't think this is the right bsc# for this action (both in the commit message and all the changes in this commit)